qutebrowser/browser/webkit/webkittab.py

```diff
@@ -704,6 +704,13 @@ def user_agent(self):
         page = self._widget.page()
         return page.userAgentForUrl(self.url())
 
+    @pyqtSlot()
+    def _on_load_started(self):
+        super()._on_load_started()
+        obj = self.networkaccessmanager()
```

Comment:
I'm sure you can think of a better name than `obj` here. `nam` for example :wink: